packages/tag/TagSelector.tsx

```diff
@@ -58,7 +59,8 @@ export class TagSelector extends Component<TagSelectorProps> {
   }
 
   private handleTagClose(event: CustomEvent) {
-    const index = this.tags.indexOf( event.detail.tag.innerHTML );
+    const target = event.target as Tag;
+    const index = this.tags.indexOf( target.label );
```

Comment:
now we can do cleaner removal:

instead of:
```ts
const index = this.tags.indexOf( target.label );
const newTags = this.tags.filter( (tag, tagIdx) => tagIdx !== index );
```
this:
```ts
const newTags = this.tags.filter( (tag) => tag !== target.label );
```